src/editor/actions/actions.js

```diff
@@ -1,30 +1,19 @@
-import debounceAction from "debounce-action";
```

Comment:
IMO we should split out the remainder of the actions in this file in seperate files to ease organization/finding.

Reply:
i 100% agree. this file is really way too big and it's very hard to use, thank you for starting that effort!

I've been thinking about this too recently, and I think that we should minimally split up action creators into files by topic, but that it might also be helpful to separate thunks and basic action-creators into separate files (and saga too, but we have just a few of these and they are already separate).

Anyway, let's continue to do this incrementally and opportunistically when we work on stuff that touches actions.

(cc @hamilton and @openjck )